
device:cuda:0, mode:age, backbone:resnet18
Traceback (most recent call last):
  File "/home/joowan/Desktop/face_pr/train_single_task.py", line 35, in <module>
    train_dl,val_dl,test_dl,criterion,optimizer,scheduler=check(mode,model)
  File "/home/joowan/Desktop/face_pr/utils/check_mode.py", line 59, in check
    train_ds=AgeDataset(phase='train')
  File "/home/joowan/Desktop/face_pr/data/datasets.py", line 191, in __init__
    self.data=json.load(f)
  File "/home/joowan/anaconda3/envs/torch/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/home/joowan/anaconda3/envs/torch/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/home/joowan/anaconda3/envs/torch/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/joowan/anaconda3/envs/torch/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)